comp/core/tagger/impl/tagger.go

```diff
@@ -458,6 +458,15 @@ func (t *TaggerWrapper) EnrichTags(tb tagset.TagsAccumulator, originInfo taggert
 			}
 		}
 	default:
+		// Disable origin detection if cardinality is none
+		// TODO: The `none` cardinality should be directly supported by the Tagger.
+		if originInfo.Cardinality == "none" {
+			originInfo.ContainerIDFromSocket = packets.NoOrigin
+			originInfo.PodUID = ""
+			originInfo.ContainerID = ""
+			return
```

Comment:
Just want to double check that the expected behavior previously was that even static global tags weren't applied? If we want the global tags, we can just break from this case and let the end of the enrich tags method apply the global tags.

Reply:
Yes, currently we want to keep a full feature parity with Legacy Origin Detection. But there is a Jira card to improve this behavior in the future 👍

Reply:
Cool thanks for the context. Legacy Origin Detection was removed just when I joined so I never really saw it.

Reply:
Technically it is still in use by everyone, but hopefully not for long 😁